etl/steps/data/meadow/lis/2023-01-18/lis_distribution.py

```diff
@@ -0,0 +1,85 @@
+"""
```

Comment:
This module is almost identical to `lis_abs_poverty`. You could wrap most of its functionality into a single function with two parameters (dependency name and target column) and put it to `shared.py` module.

Reply:
You could also have single dataset **with two tables**. That would let you have it in a single module. Do you think it would be worth it?

Reply:
Ah, the latter could work. I didn't think about it. In this case, it could be with three tables: `keyvars`, `abs_poverty` and `distribution`. I will try to do that.